src/libfuncs/int.rs

```diff
@@ -1253,15 +1293,26 @@ mod test {
 
             let result = executor.invoke_dynamic(
                 &program.funcs[0].id,
-                &[perm[0].into(), perm[1].into()],
+                &[Value::from(perm[0]), Value::from(perm[1])],
                 None,
             )?;
 
-            assert_eq!(result.builtin_stats.range_check, 1);
+            match div_rem_algorithm {
+                BoundedIntDivRemAlgorithm::KnownSmallRhs => {
+                    assert_eq!(result.builtin_stats.range_check, 3)
+                }
+                BoundedIntDivRemAlgorithm::KnownSmallQuotient { q_upper_bound: _ }
+                | BoundedIntDivRemAlgorithm::KnownSmallLhs { lhs_upper_sqrt: _ } => {
```

Comment:
You could simplify this expression like this
```suggestion
                BoundedIntDivRemAlgorithm::KnownSmallQuotient { .. }
                | BoundedIntDivRemAlgorithm::KnownSmallLhs { .. } => {
```

Reply:
Done in [`7163a9c`](https://github.com/lambdaclass/cairo_native/pull/1296/commits/7163a9c0af142574c2799b8a2e9b784da1064a93)